brave/src/test/java/com/linecorp/armeria/client/brave/BraveClientIntegrationTest.java

```diff
@@ -49,6 +53,9 @@
 @RunWith(Parameterized.class)
 public class BraveClientIntegrationTest extends ITHttpAsyncClient<WebClient> {
 
+    @Rule(order = Integer.MAX_VALUE)
```

Comment:
neat. didn't know you could override like this. We can increase upstream also. Main thing was to not hang terribly long.

Reply:
Ah, let me send a PR for that. Thanks!